frontend/README.md

```diff
@@ -0,0 +1,20 @@
+# Lumen Frontend
+
+The Lumen frontend was developed using SolidJS and TailwindCSS.
+
+## Installation
+
+Run the following commands to install the necessary packages.
+
+```
+$ cd frontend
```

Comment:
I think since this README is located inside of the `frontend` directory, we can omit the `cd frontend` command.